projects/planner/src/app/app.module.ts

```diff
@@ -1,94 +1,99 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-import { ReactiveFormsModule  } from '@angular/forms';
-import { ServiceWorkerModule } from '@angular/service-worker';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
-import { DatePipe, DecimalPipe } from '@angular/common';
+import {BrowserModule} from '@angular/platform-browser';
```

Comment:
leave it for now, but avoid such reformattings.

Reply:
Had optimize imports turned on. Turned it off, so no more surprise changes